tests/integration/deploying/declare_command_test.py

```diff
@@ -26,14 +27,12 @@ def compiled_contract_path_fixture() -> Path:
 async def test_declaring_contract(
     protostar: ProtostarFixture,
     devnet_gateway_url: str,
-    monkeypatch: pytest.MonkeyPatch,
+    alice_devnet_account: DevnetAccount,
```

Comment:
why not just `devnet_account` ? or `predefined_account`

Reply:
Renamed in [7086b29](https://github.com/software-mansion/protostar/pull/881/commits/7086b29d3a818d84585fa0c1dd81e506c5ee43cf)